contracts/ZKPVerifier.sol

```diff
@@ -71,31 +71,26 @@ contract ZKPVerifier is IZKPVerifier, Ownable {
         if (requestValidators[requestId] == ICircuitValidator(address(0x00))) {
             supportedRequests.push(requestId);
         }
-        requestQueries[requestId].valueHash = poseidon.hash(value);
-        requestQueries[requestId].operator = operator;
+        uint256 valueHash = poseidon.hash(value);
+
+        requestQueries[requestId].queryHash = poseidon.hash6(
+            [schema, slotIndex, operator, valueHash, 0, 0]
+        );
         requestQueries[requestId].circuitId = validator.getCircuitId();
-        requestQueries[requestId].slotIndex = slotIndex;
-        requestQueries[requestId].schema = schema;
         requestValidators[requestId] = validator;
         return true;
     }
 
     function setZKPRequestRaw(
         uint64 requestId,
         ICircuitValidator validator,
-        uint256 schema,
-        uint256 slotIndex,
-        uint256 operator,
-        uint256 valueHash
+        uint256 queryHash
```

Comment:
It should include also `schema`, `slotIndex`, `operator`, `valueHash` and `value`.

Reply:
fixed